templates/shared/_card--masterclass.html

```diff
@@ -10,7 +10,7 @@ <h3><a href="{{ session.Recording }}">{{ session.Topic }}</a></h3>
   </div>
   <hr class="u-no-margin--bottom">
   <div class="p-card__inner">
-    by {{ session.Owner }} on {{ session.Date }}
+    by {{ session.Owner }} on {{ <session class="Date Formatted"></session> }}
```

Comment:
Looks like autofill got a bit out of line:
```suggestion
    by {{ session.Owner }} on {{ session.Date.Formatted }}
```